crates/vm2/src/single_instruction_test/into_zk_evm.rs

```diff
@@ -91,7 +94,7 @@ pub struct MockMemory {
 // One arbitrary heap value is not enough for zk_evm
 // because it reads two U256s to read one U256.
```

Comment:
This comment is correct but seems to be in the wrong place.

Reply:
Where would you suggest to place it?

Reply:
The function below: partially_overlapping...

Looks like the comment is from a draft version where there were two values in the struct. But I ended up working around the issue with that function instead.